src/main/web/templates/handlebars/partials/json-ld/author.handlebars

```diff
@@ -0,0 +1,4 @@
+"author": {
+    "@type": "Person",
+    "name": "{{description.contact.name}}",
```

Comment:
I don't think this is valid JSON, there is a comma at the end of the last entry. Website is probably happy to show this but anything reading it I think will fail.
```suggestion
    "name": "{{description.contact.name}}"
```

Reply:
👍